examples/item.json

```diff
@@ -726,7 +725,6 @@
     "s2:not_vegetated_percentage": 0.000708,
     "s2:degraded_msi_data_percentage": 0,
     "s2:high_proba_clouds_percentage": 0.699942,
-    "s2:reflectance_conversion_factor": 1.01678413954947,
```

Comment:
Is there a different key that we should be using for this value? Feels strange to lose information and not just move it to a different key.

Reply:
No, ESA removes it without replacement. It seems not relevant to the general public.

Reply:
That's what ESA came up with in recent calls. We should probably wait for the official release of the CDSE STAC API anyway before we merge this.